examples/b2c/checkout/integrations/classes/B2CCheckInventorySample.cls

```diff
@@ -104,8 +130,7 @@ global class B2CCheckInventorySample implements sfdc_checkout.CartInventoryValid
         if (response.getStatusCode() == SuccessfulHttpRequest) {
```

Comment:
A 404 will be returned when you access: https://example.com/get-inventory. 

Add an error message for 404 that returns the following: 

"404. You must create a sample application or add your own service which returns a valid response"